### DISCUSSION TITLE
Problem with test in CI

### DISCUSSION THREAD

**Discussion by Trusted97** [phasync/phasync#27] [General]
Look a this [run](https://github.com/phasync/phasync/actions/runs/9464037361/job/26070490577#step:9:36)

I've notice that time needed for prepare test enviroment is really a lot and for this reason the test for phasync performance fail. Should we think about removing this test from the CI? Improve phasync?

**Comment by frodeborli** [comment]
Oh, it seems like that test is very strict in terms of how fast computer people must have. 

```
Failed asserting that 22281 is less than 10000.
```

The test could probably just use 30000 instead of 10000 and it would be fine.

**Reply by frodeborli** [reply]
Are you able to create an issue from this? Can I invite you to the repository?

**Reply by Trusted97** [reply]
Yes i'll create the issue from this, feel free to invite me too :)

**Comment by frodeborli** [comment]
@Trusted97 I created a discord server: https://discord.gg/GBKphgjY If you'd like to say hi and perhaps fix this with me. I made a change to the system but it still didn't fix the test. Looking into it but don't understand the CI stuff well enough atm.

**Reply by Trusted97** [reply]
I've notice now that you've fixed the problem but i'll be happy to reach you on Discord :)

**Reply by frodeborli** [reply]
@Trusted97 My username on discord is fubberman, nick frodeb

**Comment by frodeborli** [comment]
I figured it out. It is a feature, not a bug - so I fixed the test. The feature makes the framework faster for everybody, but on very slow machines this feature may mess with execution ordering simply because the main coroutine runs for too long which triggers a preempt context switch. Thank you @Trusted97 for finding this. Seems CI works now and I tagged 1.0.3.
